### DISCUSSION TITLE
Capture command output into variable

### DISCUSSION THREAD

**Discussion by M-Marton** [scandum/tintin#104] [Script help]
I would like to capture the output of a command to a variable. So It can be easily print on a split "UI". I wonder to grab the itemlist or the map to display that on a side split and also the local environment on the top split.
I find `#line capture <variable> <command>` may should do some similar thing, but I can not use it. Also I am not sure if that is limited for capturing one line or not. Is there any way to capture in a variable multiple lines of output, possibly without showing that in the main area?

Other question, is https://tintin.mudhalla.net/manual/ the latest and greatest documentation?
I was wonder if I could help to improve the documentation quality, but not really sure how. I am not a technical writer, not even native English speaker.

**Comment by scandum** [comment]
`#line capture` is used to capture output from tintin itself, for example: `#line capture test #info`.

To capture mud output you can either capture it with `#action` or you can use `#buffer` to pull the last x number of lines from the buffer when the last line of the block you want to capture is received. It's a bit finicky and you basically need to cobble something together.

The online manual is a copy of the help files provided by #help. Feel free to point out errors, but I'd like to keep it as brief as possible.
